libs/akita/src/lib/plugins/plugin.ts

```diff
@@ -71,17 +72,21 @@ export abstract class AkitaPlugin<State = any> {
   }
 
   /** This method is responsible for updating the store or one entity; it can be the whole store or one entity. */
-  protected updateStore(newState, entityId?, property?: string) {
+  protected updateStore(newState, entityId?, property?: string, replace = false) {
```

Comment:
Can you please change the params to one object? it'll not scale anymore

Reply:
yeah, I was thinking about it, but it will introduce a breaking change. is it ok?

Reply:
This is a private method. Can you see if we use it in a different package, please?

Reply:
this method is being used in custom plugins

Reply:
Ok, no need for now. Thanks